android-maps-extensions/src/main/java/com/androidmapsextensions/impl/DelegatingMarker.java

```diff
@@ -80,6 +80,7 @@ public Object getData() {
         return data;
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
I would not want users not to see deprecation of `getId`. Using this method potentially disables all the optimizations I put in dynamic marker loading feature.